### PR TITLE
chore: readme update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
# Description

Updated the repo readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,103 @@
...
-## Documentation
+## Sending events
 
-For detailed information on how to set up and use this SDK, please refer to our [documentation](https://docs.rudderstack.com/rudderstack-sdk-integration-guides/rudderstack-ruby-sdk)
+Refer to the [RudderStack Ruby SDK documentation](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-ruby-sdk/) for more information on the supported event types.
 
-### Test Queue
+## Test queue
 
 You can use the `stub` option to `Rudder::Analytics.new` to cause all requests to be stubbed, making it easier to test with this library.
```
[the remark this answers is not in the document]
Comment (pallabmaiti) [reply README.md:99]: Yes.

**Review by pallabmaiti — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply README.md:99]: is its default value `true`? https://github.com/rudderlabs/rudder-sdk-ruby/blob/ac142d2f8746e2428f157b162a027eed9bcd8447/spec/rudder/analytics_spec.rb#L8 referencing this. @pallabmaiti

[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply README.md:99]: I was wondering if we need this sentence in the readme, we don't have it included in the Ruby docs.

Comment (ameypv-rudder) [inline README.md:99]:
```suggestion
Enable the `stub` option while initializing the SDK to stub all the requests, making it easier for you to test with this library.
```

**Pull request merged**